packages/client/src/pages/Admin/admin.tsx

```diff
@@ -0,0 +1,12 @@
+import { Navbar } from '@pages/Admin/navbar';
+import { Box } from '@mui/material';
+import { Outlet } from 'react-router-dom';
+
+export const Admin = () => {
+  return (
+    <Box component="div" sx={{ display: 'flex' }}>
```

Comment:
```suggestion
    <Box sx={{ display: 'flex' }}>
```
You don't need to specify div here